lib/Store.php

```diff
@@ -32,13 +32,19 @@ protected function __construct(array &$config)
      *
      * This function checks whether a given user has been enabled for WebAuthn.
      *
-     * @param string $userId        The hash identifying the user at an IdP.
-     * @param bool   $defaultIfNx   if not found in the DB, should the user be considered enabled (true)
+     * @param string $userId The hash identifying the user at an IdP.
+     * @param bool $toggle variable which is associated with $force because it determines its meaning, it either
+     *                     simply means whether to trigger webauthn authentication or switch the default settings,
+     *                     if null (was not sent as attribute) then the information from database is used
+     * @param bool $defaultIfNx if not found in the DB, should the user be considered enabled (true)
      *                              or disabled(false)
+     * @param bool $force switch that determines how $toggle will be used, if true then value of $toggle
+     *                    will mean whether to trigger (true) or not (false) the webauthn authentication,
+     *                    if false then $toggle means whether to switch the value of $defaultEnabled and then use that
      *
      * @return bool True if the user is enabled for 2FA, false if not
      */
-    abstract public function is2FAEnabled(string $userId, bool $defaultIfNx): bool;
+    abstract public function is2FAEnabled(string $userId, $toggle, bool $defaultIfNx, bool $force): bool;
```

Comment:
When you add arguments, ideally add them at the end with defaults to previous behavior (and add missing type hint), so probably `string $userId, bool $defaultIfNx, bool $toggle = false, bool $force = false`